en/docs/get-started/try-out-flow.md

```diff
@@ -88,8 +88,8 @@ The Accredited Data Recipient application redirects the bank customer to authent
       "typ": "JWT"
     }
     {
-      "aud": "https://localhost:9446/oauth2/token",
-      "iss": "3nzwb1VuaUISzujNe6QjDxlgnCka",
+      "aud": "<This is the audience that the ID token is intended for. Example, https://<IS_HOST>:9446/oauth2/token>",
```

Comment:
do the same changes in the try-out/account-and-transaction-flow/ doc.

Reply:
Merging this PR as other changes will be addressed in https://github.com/wso2/docs-ob-toolkit-cds/pull/16